.github/workflows/linter.yml

```
@@ -8,18 +8,13 @@ jobs:
-          bin/custom-gcl
-        key: ${{ runner.os }}-${{ runner.arch }}-tools-${{ hashFiles('.custom-gcl.yml', 'tools/**/go.mod', 'tools/**/go.sum', 'tools/**/*.go') }}
-    - run: script/lint.sh
-      env:
-        CHECK_GITHUB_OPENAPI: 1
-        GITHUB_TOKEN: ${{ github.token }}
+      - uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6.0.2
+      - uses: actions/setup-go@4b73464bb391d4059bd26b0524d20df3927bd417 # v6.3.0
+        with:
+          go-version: stable
+          cache-dependency-path: "**/go.sum"
+      - name: Setup golangci-lint ${{ matrix.modules }}
```

actionlint (1.7.11)

[error] property "modules" is not defined in object type {}

(expression)